setup.py

```diff
@@ -1,26 +1,3 @@
 from setuptools import setup
 
-from batchpr import __version__
-
-with open('README.rst') as infile:
-    long_description = infile.read()
-
-setup(
-    version=__version__,
-    url="https://github.com/astrofrog/batchpr",
-    name="batchpr",
-    description='Easy automated pull requests to GitHub repositories',
-    long_description=long_description,
-    packages=['batchpr'],
-    package_data={'pytest_mpl': ['classic.mplstyle']},
```

Comment:
I don't see why the `package_data` is needed. Is it okay to remove?

Reply:
Yeah, that's leftover from another package